test/spec/modes/add_note.js

```diff
@@ -1,4 +1,4 @@
-describe('iD.modeAddNote', function() {
+describe.skip('iD.modeAddNote', function() {
```

Comment:
Just checking, is this by design? It does skip the whole block, right? (https://vitest.dev/api/#describe-skip)

Reply:
The actual test code in this file is all commented out (see line 28 and following), so skipping the whole thing saves a little bit of setup time. I haven't looked into why the tests are commented out for now, but that would be something for a follow-up PR anyway.